recipes/cwinpy/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "cwinpy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3caedf3aac6b8ef9bc0df8467e1610587c1fae3b2a0266ee6c839d45a79e5cb0
+
+build:
+  entry_points:
+    - cwinpy_knope = cwinpy.knope.knope:knope_cli
+    - cwinpy_knope_dag = cwinpy.knope.knope:knope_dag_cli
+    - cwinpy_knope_generate_pp_plots = cwinpy.knope.testing:generate_pp_plots
+  number: 0
+  skip: true  # [win]
```

Comment:
You can remove the `skip` for windows

```suggestion
```

Once merged, the feedstock will only build on linux, we can just ignore the windows failures here. Users you attempt to install this package on a windows machine will be presented with a relevant error explaining that the dependencies are not available.